src/hotspot/share/runtime/globals.hpp

```diff
@@ -1982,7 +1982,7 @@ const int ObjectAlignmentInBytes = 8;
   product(int, LockingMode, LM_LEGACY, EXPERIMENTAL,                        \
           "Select locking mode: "                                           \
           "0: monitors only, "                                              \
-          "1: monitors & traditional stack-locking (default), "             \
+          "1: monitors & legacy stack-locking (default), "                  \
           "2: monitors & new lightweight locking")                          \
```

Comment:
Can we include the `LM_XXX` values in the description string so it is clear which maps to what.